### PR TITLE
Add documentation for telemetry

### PR DESCRIPTION
**Pull request by amytai**
Added appropriate references throughout existing documentation for
telemetry. Includes description of the new API function as well as
instructions for turning the functionality off.

### CODE REVIEW

File: api.md [hunk 1]
```diff
@@ -712,6 +713,19 @@ SELECT device_id, first(temp, time)
 
 ---
 
+## get_report() [](get_report)
+
+The `get_report()` function prints out the JSON that is sent to our servers if
+background telemetry is enabled. It takes no arguments.
```
Comment (dianasaur323) [inline api.md]: Should we add a link here to telemetry?

File: using-timescaledb/telemetry.md [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+		“PostGIS”:true
+    	}
+}
+```
+Specifically, both UUID fields are randomly generated by appropriately seeded random
+number generators, so there is no identification information in this field. For full
+transparency, we expose a new API function, [`get_report`], that prints out the exact JSON that is sent to our servers.
+
+## Version Checking
+In response to the telemetry report, users will automatically receive the most recent version of Timescale available for installation. This version will be recorded in the user’s Timescale logs, along with any applicable out-of-date version warnings. While you do not have to update immediately to the newest release, many users have reported that performance issues or bugs automatically resolve after updating their version of Timescale.
```
Comment (dianasaur323) [inline using-timescaledb/telemetry.md]: I "think" we want to say TimescaleDB here

File: using-timescaledb/telemetry.md [hunk 3]
```diff
@@ -0,0 +1,52 @@
+# Telemetry and Version Checking
```
Comment (dianasaur323) [inline using-timescaledb/telemetry.md]: In this section, could we include the config file that you can manually change? This is to allow users who want to turn off the flag even before they CREATE EXTENSION.

File: api.md (hunk 1 above)
Comment (amytai) [reply api.md]: done

File: using-timescaledb/telemetry.md (hunk 2 above)
Comment (amytai) [reply using-timescaledb/telemetry.md]: Yeah fixed.

File: using-timescaledb/telemetry.md (hunk 3 above)
Comment (amytai) [reply using-timescaledb/telemetry.md]: Reorganized this section after discussing w Sven

File: getting-started/setup.md [hunk 4]
```diff
@@ -48,7 +48,13 @@ data.
 storing time-series data in a PostgreSQL database, and want to move this data
 to a TimescaleDB hypertable.
 
+---
+
+Note: Starting in v0.12.0, TimescaleDB will by default enable [telemetry
+reporting].
```
[the remark this answers is not in the document]
Comment (amytai) [reply getting-started/setup.md]: done

File: api.md [hunk 5]
```diff
@@ -712,6 +713,19 @@ SELECT device_id, first(temp, time)
 
 ---
 
+## get_report() [](get_report)
+
+The `get_report()` function prints out the JSON that is sent to our servers if
+background [telemetry] is enabled. It takes no arguments.
+
+#### Example
```
[the remark this answers is not in the document]
Comment (amytai) [reply api.md]: done

File: api.md [hunk 6]
```diff
@@ -712,6 +713,19 @@ SELECT device_id, first(temp, time)
 
 ---
 
+## get_report() [](get_report)
+
+The `get_report()` function prints out the JSON that is sent to our servers if
+background [telemetry] is enabled. It takes no arguments.
```
[the remark this answers is not in the document]
Comment (amytai) [reply api.md]: Ah ok, I didn't want to invoke the name of the function, but some of the entries do this so I wasn't positive.

File: api.md [hunk 7]
```diff
@@ -712,6 +713,19 @@ SELECT device_id, first(temp, time)
 
 ---
 
+## get_report() [](get_report)
```
[the remark this answers is not in the document]
Comment (amytai) [reply api.md]: Yep, didn't see the Utilities section. Moved.

File: using-timescaledb/telemetry.md [hunk 8]
```diff
@@ -0,0 +1,68 @@
...
+timescaledb.metrics_level=default
+```
+
+or run the following command in psql:
+
+```
+ALTER [SYSTEM | DATABASE | USER] SET timescaledb.metrics_level=default
+```
+
+[get_report]: /api/#get_report
```
[the remark this answers is not in the document]
Comment (amytai) [reply using-timescaledb/telemetry.md]: done

File: using-timescaledb/telemetry.md [hunk 9]
```diff
@@ -0,0 +1,68 @@
...
+    	{
+		"pg_prometheus":false,
+		"PostGIS":true
+    	}
+}
+```
+
+Specifically, both UUID fields are randomly generated by appropriately seeded random
+number generators, so there is no identification information in this field. For full
+transparency, we expose a new API function, [`get_report`], that prints out the exact JSON that is sent to our servers.
```
[the remark this answers is not in the document]
Comment (amytai) [reply using-timescaledb/telemetry.md]: done

File: using-timescaledb/telemetry.md [hunk 10]
```diff
@@ -0,0 +1,68 @@
...
+
+## Version Checking
+In response to the telemetry report, users will automatically receive the most recent version of TimescaleDB
```
[the remark this answers is not in the document]
Comment (amytai) [reply using-timescaledb/telemetry.md]: done

File: using-timescaledb/telemetry.md [hunk 11]
```diff
@@ -0,0 +1,68 @@
...
+available for installation. This version will be recorded in the user’s server logs, along with any applicable
+out-of-date version warnings. While you do not have to update immediately to the newest release, many users have
+reported that performance issues or bugs automatically resolve after updating their version of TimescaleDB. 
+
+## Disabling Telemetry
+Although we would like to invite our community to help us keep improving our product, we do understand when users would like to disable it. Note that disabling telemetry also disables the version checking functionality.
```
[the remark this answers is not in the document]
Comment (amytai) [reply using-timescaledb/telemetry.md]: Fixed

File: using-timescaledb/update-db.md [hunk 12]
```diff
@@ -57,6 +57,10 @@ After executing the command, the psql `\dx` command should show the latest versi
 (1 row)
 ```
 
+>:TIP: Beginning in v0.12.0, [telemetry] reporting will also enable automatic
```
[the remark this answers is not in the document]
Comment (amytai) [reply using-timescaledb/update-db.md]: done

**Pull request merged**